src/memory_pool.c

```diff
@@ -168,6 +209,10 @@ umf_result_t umfPoolGetMemoryProvider(umf_memory_pool_handle_t hPool,
     return UMF_RESULT_SUCCESS;
 }
 
+const char *umfPoolGetName(umf_memory_pool_handle_t hPool) {
+    return hPool->ops.get_name(hPool->pool_priv);
```

Comment:
missing check for empty hPool + add a negative test

Reply:
Done